Cargo.toml

```diff
@@ -1,3 +1,8 @@
+[profile.stripdebuginfo]
+# Used by CI to lower binary size for test artifacts.
+inherits = "test"
+strip = "debuginfo"
+
```

Comment:
This has to come at the top, because otherwise the `dep-sort` thinks its an out-of-order dependency..